ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -142,11 +142,15 @@ l3leaf:
           id: 4
           mgmt_ip: 192.168.200.108/24
           mac_address: '0c:1d:c0:1d:62:01'
+          mlag_peer_ipv4_address: 192.168.44.41
+          mlag_peer_l3_ipv4_address: 192.168.44.41
```

Comment:
let's not use the same address for non l3 and l3 in the test to avoid confusion in the results (like 192.168.45.1 for the next one)

Reply:
Also the proper `192.168.44.40/31` and `192.168.44.41/31` rather than 41 and 42 if we want it to be "networkingly" correct

Reply:
Maybe we could just expose `mlag_peer_ipv4_subnet` and always assign the first and second host address of the given subnet. That would make it easier to use group inheritance etc.